tests/Microsoft.Identity.Test.Integration/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -57,19 +57,47 @@ public void TestInitialize()
 
         #region Happy Path Tests
         [TestMethod]
-        public async Task AcquireTokenWithManagedUsernamePasswordAsync()
+        public async Task ROPC_AAD_Async()
         {
             var labResponse = LabUserHelper.GetDefaultUser();
             await RunHappyPathTestAsync(labResponse).ConfigureAwait(false);
         }
 
         [TestMethod]
-        public async Task AcquireTokenWithFederatedUsernamePasswordAsync()
+        public async Task ROPC_ADFSv4Federated_Async()
         {
             var labResponse = LabUserHelper.GetAdfsUser(FederationProvider.AdfsV4, true);
             await RunHappyPathTestAsync(labResponse).ConfigureAwait(false);
         }
 
+        [TestMethod]
+        public async Task ROPC_ADFSv4Managed_Async()
```

Comment:
what does ROPC stand for?

Reply:
Resource Owner Password Credential. It's the name of the flow in the OAuth2 spec.